Makefile

```diff
@@ -161,26 +167,33 @@ test-typecheck-build-support: ## Typecheck build-support Python code
 test-typecheck-pants: test-typecheck-pulumi test-typecheck-build-support ## Typecheck Python code with Pants
 
 .PHONY: test-integration
-test-integration: export COMPOSE_IGNORE_ORPHANS=1
 test-integration: build-test-integration ## Build and run integration tests
-	# Usage:
-	# make test-integration
-	# make test-integration TARGET=grapl-analyzerlib-integration-tests
-	$(WITH_LOCAL_GRAPL_ENV) \
-	$(MAKE) up-detach PROJECT_NAME="grapl-integration_tests" && \
-	test/docker-compose-with-error.sh \
-		-f ./test/docker-compose.integration-tests.yml \
-		-p "grapl-integration_tests" \
-		-t "$(TARGET)"
+	$(MAKE) test-with-env \
+		TEST_FILE="./test/docker-compose.integration-tests.yml" \
+		COMPOSE_PROJECT_NAME="grapl-integration_tests"
```

Comment:
Reading this initially, it's a bit surprising that the `TEST_FILE` isn't handled by `COMPOSE_FILE`, and that it and `COMPOSE_PROJECT_NAME` aren't supplied as target-specific variables, as they are elsewhere.

That leads me to assume that `test-with-env` is doing something "special"... if that special-ness isn't truly required, it would be nice to eliminate it; if it is, it would be good to explicitly call out the reasons for it.

Reply:
Can't set `COMPOSE_FILE` for the whole target here because we we're executing two docker-compose with different files.

Reply:
For `COMPOSE_PROJECT_NAME` I'll change to set that as other targets do. For `TEST_FILE` I didn't want to set `COMPOSE_FILE` because we're executing docker-compose without that file and I wanted to be more explicit about that. I wouldn't want that value to interfere with that execution.